Unbreak on ILP32 archs;

/pobj/kf6-kfilemetadata-6.17.0/kfilemetadata-6.17.0/src/extractors/plaintextextractor.cpp:163:45: error: non-constant-expression cannot be narrowed from
type 'qint64' (aka 'long long') to 'qsizetype' (aka 'int') in initializer list [-Wc++11-narrowing]

From FreeBSD:
https://cgit.freebsd.org/ports/commit/?id=69f4a3e0060cce7a59fe05f726e36d0d5ca80027

Index: src/extractors/plaintextextractor.cpp
--- src/extractors/plaintextextractor.cpp.orig
+++ src/extractors/plaintextextractor.cpp
@@ -119,7 +119,7 @@ void PlainTextExtractor::extract(ExtractionResult* res
 
     // Read the first chunk, detect the encoding and decode it
     QByteArray chunk(256 * 1024, Qt::Uninitialized);
-    auto size = file.read(chunk.data(), chunk.size());
+    auto size = static_cast<qsizetype>(file.read(chunk.data(), chunk.size()));
 
     QStringDecoder codec{autodetectCodec({chunk.data(), size})};
 
@@ -151,7 +151,7 @@ void PlainTextExtractor::extract(ExtractionResult* res
 
     // Read and decode the remainder
     while (!file.atEnd()) {
-        auto size = file.read(chunk.data(), chunk.size());
+        auto size = static_cast<qsizetype>(file.read(chunk.data(), chunk.size()));
         if (size < 0) {
             // may happen when the file is truncated during read
             qCWarning(KFILEMETADATA_LOG) << "Error reading" << result->inputUrl();
